config/jest/cssTransform.js

```diff
@@ -5,7 +5,9 @@
 
 module.exports = {
   process() {
-    return 'module.exports = {};';
+    return {
```

Comment:
the shape of the return slightly changed, I followed this guide: https://web.archive.org/web/20230330085721/https://jestjs.io/docs/28.x/upgrading-to-jest28#transformer